salt/preflight/mandatory.sls

```diff
@@ -0,0 +1,45 @@
+{% import_yaml 'preflight/defaults.yaml' as settings %}
+
+install mandatory packages:
+  pkg.installed:
+    - pkgs: {{ settings.mandatory.packages }}
+
+{%- for directory in settings.mandatory.available_pathdirs %}
+make sure {{ directory }} does not exist:
+  file.absent:
+    - name: {{ directory }}
+{%- endfor %}
+
+{%- for service in settings.mandatory.services.running %}
+enable {{ service  }} service:
+  service.running:
+    - name: {{ service  }}
+    - enable: true
+{%- endfor %}
+
+add the module br_netfilter to kernel:    # required to set sysctl net.bridge.bridge-nf-call-iptables
+  kmod.present:
+    - name: br_netfilter
+
+{%- for item, value in settings.mandatory.sysctl_values.items() %}
+set sysctl {{ item }} value to {{ value }}:
+  sysctl.present:
+    - name: {{ item }}
+    - value: {{ value }}
+    - config: /etc/sysctl.d/60-metalk8s.conf
+    {%- if item in ("net.bridge.bridge-nf-call-ip6tables", "net.bridge.bridge-nf-call-iptables") %}
+    - require:
+      - kmod: add the module br_netfilter to kernel
+    {%- endif %}
+{%- endfor %}
+
+disable devices and files for paging and swapping:
+  cmd.run:
+    - name: swapoff -a
```

Comment:
This would not persist across reboots.

Also, we need to fine-tune our story w.r.t. swap, there was some disagreement with TS about this.